Sprint_2 Version 1

Ruurdbk opened the pull request:
Efficiency statistics: Includes an option which shows efficiency statistics of the NED2XYZ conversion. Version 1 shows shows total time required to convert and soundings converted per second. Version 2 is identical but makes use of functions.

Ruurdbk committed:
Sprint_2 Version 1

Efficiency statistics: Includes a new library (timer) which allows to calculate time to convert a file and speed of conversion.

Ruurdbk committed:
Sprint_2 Version 2

boelek commented:
Works fine, but I noticed one problem. The stats only take one file and if there are multiple NED files this will not work correctly. Try to change the stats in such a way that they include all the files. I suggest you count the soundings in the NED2XYZ function.

Ruurdbk committed:
Sprint_2_Final_Version

Final version of Sprint 2. This version converts and show efficiency statistics for all the files stored in the directory instead of a single file.

boelek commented:
Works like a charm. This time all the soundings were counted. This concludes the functionality of sprint 2. Thanks :-)
